overlord/devicestate/export_test.go

```diff
@@ -544,6 +554,8 @@ func CheckEncryptionSetupDataFromCache(st *state.State, label string) error {
 }
 
 func CleanUpEncryptionSetupDataInCache(st *state.State, label string) {
+	st.Lock()
+	defer st.Unlock()
 	key := encryptionSetupDataKey{label}
```

Comment:
this is a bit unusual, also because the Set side doesn't do this

Reply:
Good point, I've tweaked the tests a little bit so locking happens also in `MockEncryptionSetupDataInCache`, as in the end accessing to the cache does require the lock.